src/Auth/Properties/launchSettings.json

```
@@ -0,0 +1,11 @@
+{
+  "profiles": {
+    "Auth": {
+      "commandName": "Project",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      },
+      "applicationUrl": "http://localhost:5000"
```

```suggestion
      "applicationUrl": "https://localhost:5000"
```

There are a few other places where this needs to be changed.

---

I made more complicated ports but made it work: 3aa6174d